webapp/src/app.tsx

```diff
@@ -774,7 +774,7 @@ export class ProjectView
     }
 
     private pickEditorFor(f: pkg.File): srceditor.Editor {
-        return this.allEditors.filter(e => e.acceptsFile(f))[0]
+        return this.allEditors.filter(e => e.acceptsFile(f), 1)[0]
```

Comment:
This passes `1` as `this` to the callback. Not very useful?

Reply:
wrong very wrong